### PR TITLE
Convert cli-guides to be a single repo project

### PR DESCRIPTION
**Pull request by mansona**
Howdy folks 👋 

When we set up the cli-guides-source it was using a very early version of Guidemaker that **required** the source and the app to be in different repos. This is because of the fact that Guidemaker is essentially an extraction of the tech behind the Ember Guides App and that was one of the original design principles of that project.

Having different repos was useful in some regards but caused a number of issues:

- we are not able to preview pull requests in the "source only" repo. This is because the preview can only happen after a new version of the source has been released
- deploying a new version of the guides to production requires deploying a markdown-only npm package and updating that dependency on the consuming app
- seeing your changes locally requires a linking process that may be confusing to new contributors

This PR solves all of these problems, and as a bonus you will be able to see a netlify demo of this PR to show you what it looks like 🎉 

I am happy to discuss this some more if anyone has any questions 👍

### CODE REVIEW

**Comment by mansona** [comment]
For anyone who wants to have a look at the deploy-preview it's here: https://deploy-preview-28--ember-cli-guide.netlify.com

I think the name of the app was changed since the build notification was added ☝️

**Comment by Gaurav0** [comment]
Also, let's update the README.md with the new simple instructions to running the app.

**Comment by mansona** [comment]
@Gaurav0 I have updated the README with the simpler instructions 👍 

@locks nice catch 🎉 the right move here was to remove the title from that document because it's handled by guidemaker internals (because it needs to add the current page to the title dynamically) 👍

File: README.md [hunk 1]
```diff
@@ -20,21 +20,23 @@ You will need the following things properly installed on your computer.
 * [Node.js](https://nodejs.org/) (with npm)
 * [Ember CLI](https://ember-cli.com/)
 * [Google Chrome](https://google.com/chrome/)
-* [npm](https://docs.npmjs.com/cli/install)
 
 ## Local Development
 
-To see what a local copy of the Guides markdown looks like:
+The Ember-Cli Guides is just a normal Ember application so if you want to run it locally you follow the standard steps:
```
Comment (Gaurav0) [inline README.md]: let's remove 'just'

Comment (mansona) [reply README.md]: updated 👍

**Pull request merged**